packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -49,6 +62,15 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
   @visibleForTesting
   final FileSystem fs;
 
+  /// A wrapper for the [dart:io.Platform] API.
+  ///
+  /// This is useful in tests, where the system platform (the default) can be
```

Comment:
I don't understand the use of this. Even if the test pretends to be on Linux while it's actually running on Mac, we need to skip the golden check, no?